real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/ui/PercentileLayoutManger.java

```diff
@@ -0,0 +1,55 @@
+package com.roboclub.robobuggy.ui;
+
+import java.awt.Component;
+import java.awt.Container;
+import java.awt.Dimension;
+import java.awt.LayoutManager;
+import java.util.ArrayList;
+
+
+class PercentileLayoutManger implements LayoutManager{
+	ArrayList<ComponentData> components;
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://www.codacy.com/app/Mr-Trevor-Decker/RoboBuggy/file/1801801914/issues/source?bid=2938665&fileBranchId=3059924#l11)